system_files/bluefin/etc/bazaar/hooks.py

```
@@ -26,16 +26,161 @@ def spawn_and_detach(args):
+                return 'ok'
+            else:
+                return 'abort'
+
+        case 'catch':
+            return 'abort'
+
+        case 'action':
+            try:
+                cask = casks[transaction_appid]
+                spawn_brew_tap_cask(tap, cask)
+            except:
```

Ruff (0.15.20)

[error] Do not use bare `except`

(E722)